docs/pages/tutorial/initialize-app.md

```diff
@@ -0,0 +1,94 @@
+
+### Alternate: manually initialize with `create-expo-app`
+
+If you prefer to create everything from scratch, you can use the `create-expo-app` command to initialize a new Expo project.
+
+In the directory of your choice with your preferred terminal, run the following command to initialize a new project:
+
+<Terminal cmd={[
+'# Create a project named StickerSmash',
+'$ npx create-expo-app StickerSmash',
+'',
+'# Navigate to the project directory',
+'$ cd StickerSmash'
+]} cmdCopy="npx create-expo-app StickerSmash && cd StickerSmash" />
+
+This command will create a new directory for the project with the name: **StickerSmash**.
+
+## Step 2: Run the app on mobile
+
+When you have initialized the app, navigate inside the project directory in your terminal window.
+
+After navigating inside the project directory, run the following command to trigger the [development server](/guides/how-expo-works/#expo-development-server) and see the bare-bones app in action:
+
+<Terminal cmd={['$ npx expo start']} />
+
+After starting the development server, you can open the project in the Expo Go app on your device. Learn more on [how to open the app when developing with Expo on a physical device](/get-started/create-a-new-app/#opening-the-app-on-your-phonetablet).
+
+Here is how the app in the current state, running on an iOS simulator and a physical Android device:
+
+<ImageSpotlight alt="App running on an iOS simulator and a physical Android device." src="/static/images/tutorial/app-running-on-mobile-platforms.jpg" style={{maxWidth: 480}} />
+
+The text displayed on the app's screen above can be found in the **App.js** file of the initialized project. It is the entry point of the app, and is executed when the app starts.
+
+## Step 3: Install dependencies for the web
+
+When you create a new project with `create-expo-app`, it uses the default [blank template](https://github.com/expo/expo/tree/main/templates/expo-template-blank) that comes pre-configured with necessary dependencies to run your app on mobile platforms. To support the web, you need to add a few additional dependencies to the project. In a terminal window, run the following command to install the necessary dependencies to include web support:
+
+<Terminal cmd={['$ npx expo install react-native-web@~0.18.7 react-dom@18.0.0 @expo/webpack-config@^0.17.0']} />
+
+> If you have initialized your app with the starter template, you can skip this step. These dependencies are already installed in the template.
+
+Before running the above installation command, make sure to terminate `npx expo start` if it is already running the development server.
+
+## Step 4: Run the app on the web
+
+After installing the dependencies, run the development server again using `npx expo start`. To see the web app in action, press <kbd>W</kbd> in the terminal window. It will open the web app in your default web browser at a specific URL.
```

Comment:
```suggestion
After installing the dependencies, run the development server again using `npx expo start`. To see the web app in action, press <kbd>w</kbd> in the terminal window. It will open the web app in your default web browser at a specific URL.
```